### ISSUE TITLE
Error message doesn't really match error for missing column

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#471]
I got this error message in Lightdash...
![image](https://user-images.githubusercontent.com/31848148/134678755-703865c7-42e8-4104-b4b6-0722761fb5bd.png)

When I took a look at the tables in BigQuery, the column `labels` didn't exist...(here's the schema for the `base_github_pull_requests`)
![image](https://user-images.githubusercontent.com/31848148/134678728-4eced063-8e02-4f7f-bc47-81e1b27163ab.png)

_**but**_ when I look at the schema file for `base_github_pull_requests`, there's a column defined in there called `labels`
![image](https://user-images.githubusercontent.com/31848148/134678881-ce57f3de-6ada-4412-accc-ee0dd72b7d38.png)

Basically, `labels` is a column that used to exist in the model, but it doesn't anymore. dbt doesn't fail if you've defined a column in your schema file that doesn't exist in the table. But, Lightdash does. 

I'm not sure if there's a way of catching this weird scenario, but I think this could use a better error message!

E.g. `The dimension "labels" in your model's schema doesn't exist in your target warehouse.`

**Comment by TuringLovesDeathMetal** [comment]
closing this - no longer relevant
